### PR TITLE
Clean up links and blurb about security with clients

### PR DESCRIPTION
**Pull request by lukebakken**

### CODE REVIEW

File: content/riak/kv/2.0.0/using/security.md [hunk 1]
```diff
@@ -158,59 +158,25 @@ over the Internet. This allows us to open a dialogue with the security
...
 For instructions on how to apply permissions and to require client
-authentication, please see our documentation on [Riak Security](/riak/kv/2.0.0/using/security/basics).
+authentication, please see our documentation on [Riak Security][security basics].
 
 ### Network Configurations
 
 Being a distributed database means that much of Riak's security springs
 from how you configure your network. We have a few recommendations for
-[Security and Firewalls](/riak/kv/2.0.0/using/security/basics).
+[Security and Firewalls][security basics].
```
Comment (laurenrother) [inline content/riak/kv/2.0.0/using/security.md]: Where is [security basics]? 

In order for  [security basics] to work, there needs to be (below the metatdata but above the text of the doc) something that looks like: 
[security basics]: /riak/kv/2.2.0/using/security/basics

Then you can plunk the [security basics] reference in the doc wherever you please! :)

File: content/riak/kv/2.0.1/using/security.md [hunk 2]
```diff
@@ -158,59 +158,25 @@ over the Internet. This allows us to open a dialogue with the security
...
------END PGP PUBLIC KEY BLOCK-----
-</tt></pre>
-
 ## Security Best Practices
 
 ### Authentication and Authorization
 
 For instructions on how to apply permissions and to require client
-authentication, please see our documentation on [Riak Security](/riak/kv/2.0.1/using/security/basics).
+authentication, please see our documentation on [Riak Security][security basics].
```
Comment (laurenrother) [inline content/riak/kv/2.0.1/using/security.md]: Same comment as above applies here.

File: content/riak/kv/2.0.0/using/security.md (hunk 1 above)
Comment (lukebakken) [inline content/riak/kv/2.0.0/using/security.md]: I'll get it. The `2.1.4` version of this doc must have it at the bottom. I figured it magically worked somehow 😸

Comment (lukebakken) [inline content/riak/kv/2.0.0/using/security.md]: What about this document?

https://raw.githubusercontent.com/basho/basho_docs/master/content/riak/kv/2.0.0/_reference-links.md

**Pull request merged**